wikidata_game/stage-game.php

```diff
@@ -0,0 +1,47 @@
+<?php
+    if(!isset($_SERVER['HTTP_X_GITHUB_EVENT'])){
+        exit;
+    }
+
+    if($_SERVER['HTTP_X_GITHUB_EVENT'] != 'pull_request'){
```

Comment:
We should probably use https://developer.github.com/webhooks/securing/ to make sure it's only GitHub who is able to usefully able to call this file